src/generate_tangent.jl

```diff
@@ -0,0 +1,13 @@
+rand_tangent(rng::AbstractRNG, x::Integer) = Zero()
+
+rand_tangent(rng::AbstractRNG, x::T) where {T<:Number} = randn(rng, T)
```

Comment:
I wonder if this makes it too zero centered?

Should we make if offzero?
```suggestion
rand_tangent(rng::AbstractRNG, x::T) where {T<:Number} = 100randn(rng, T) + 10rand(rng, T)
```

Reply:
I've never found this to be a problem. Generally speaking low-variance values aren't a bad bet because large values tend to stress out numerics. I mean, obviously you can produce bad numerics with what I've proposed, but I would expect high variance stuff to produce more problems than it solves tbh.

Reply:
Ok fair enough